play/src/lib/HelperFunctions.tsx

```diff
@@ -296,8 +296,10 @@ export const handleNormalizeAnswers = (currentContents: any) => { // eslint-disa
           //  reducing case and removing characters
 
           // this extracts numeric values from a string and adds them to the normalized text array.
```

Comment:
Adjustments here are made to handle conditions with %, that arose in testing.

Given two inputs of '5%' and '50%', our original algorithm would extract the 5 and the 50, respectively, leaving '%' and '%', which would then mistakenly produce equality. Now we cut that out and rely on a string comparison later in the code to handle any matching percentages.